pygeoapi/flask_app.py

```diff
@@ -155,6 +155,36 @@ def conformance():
     return get_response(api_.conformance(request))
 
 
+@BLUEPRINT.route('/TileMatrixSets/WorldCRS84Quad')
+def WorldCRS84Quad():
```

Comment:
with a dynamic approach the name could be `def get_tilematrix_set(tileMatrixSetId=None):`